api/api.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"github.com/RTradeLtd/Temporal/api/middleware"
 	"github.com/RTradeLtd/Temporal/database"
+	"github.com/RTradeLtd/Temporal/eh"
```

Comment:
Very interesting choice of package name but... i guess it's fine haha, and doenst have any namespace conflicts

Reply:
Haha, it was `eh` or `errorhandler`. Too lazy to alias the import to `eh` so this seemed better :joy: